_pages/business-units/18f/projects-partners/leading-projects.md

```diff
@@ -30,17 +30,54 @@ Someone (the partner, 18F leadership, or others) might ask you about how the pro
 
 Teams are more resilient and equitable when administrative tasks like scheduling meetings or taking notes are shared. It can help to proactively set up systems to rotate this work (for example: a schedule for who will send weekly ship emails each week).
 
+### Project coordination checklist
+For most 18F projects, these are generally the tasks that need to be done. **Typically, the team will share the load on doing this work** — it’s the project lead’s role to ensure these things are happening.
+
+#### Beginning of Project
+* Facilitate creation of a [team charter](https://docs.google.com/document/d/1J8MqOA-JkLKzSFTVxeXDQub7ZTsWuNq5nOZ_ACICdig/edit?usp=sharing) that includes how decisions will be made and how the team will navigate disagreement
+  * Recommended: Discuss how the team will navigate disagreement
+* Identify stakeholder accessibility needs for project work (in collaboration tools and deliverables)
+* Ensure sprint planning, review, and other recurring meetings are scheduled
+* Facilitate identification of project management tracking tools
+* Facilitate identification of how administrative tasks will be rotated
+* Create and maintain a [Project Readme](https://docs.google.com/document/d/1_vL462nws0l1WXMTyTdWcmU2GyjHzF88dHvnLIvYeq4/edit?usp=sharing) in the project’s Drive folder (Account Manager will establish project folder)
+
+#### Ongoing
+* Foster a safe and inclusive environment
+* Identify the team’s approach to stakeholder management and who serves as primary contact for whom
+* Facilitate review of whether promised deliverable are feasible
+* Facilitate scheduling of the work the team decides needs to be done
+* Assess progress against plan
+* Identify the level of fidelity/polish stakeholders need in deliverables (who are they sharing with, what's the intended shelf-life)
+* Weekly:
+  * Meet with account manager
+  * Monitor project burn doc with account manager
+  * Compile and post a Weekly Ship; send to partner and post in [#the-shipping-news](https://gsa-tts.slack.com/archives/C4HGPF9QA)
+* Every two weeks: Complete [Project Health Assessment](https://airtable.com/shrvaV9rDyifdSUbA) with team
+* Ensure midpoint and other higher-profile presentations are rehearsed or otherwise well prepared
+* Monthly: Conduct an internal team retrospective
+* Work with account manager and staffing team when project team identifies a need for a shift in staffing
+* Onboard new team members to the project, as needed, including revisiting team charter with full team
+* Periodically: Check in on equitable distribution of administrative tasks among team members: re-balance as necessary
+* Add to and organize the project’s Drive folder
+* [Get support]((#escalation-paths-for-issues-that-may-arise)) in cases where team has trouble resolving disagreement or with other project issues
```

Comment:
CircleCI is failing because this link doesn't exist. Did the heading change, maybe?